### PR TITLE
Ia 977 tests fail when polio plugin is not activated

### PR DESCRIPTION
**Pull request by SimonDus**

### CODE REVIEW

**Comment by SimonDus** [comment]
This is a workaround. As Polio is a separate project and while running test unit test is looking the whole project I think the solution of skipping polio test if polio is not loaded is the most straightforward way to do it as it's kinda useless to test a plugin which is not installed / loaded.

File: hat/settings.py [hunk 1]
```diff
@@ -264,12 +263,15 @@ def is_superuser(u):
 LOGIN_URL = "/login"
 LOGIN_REDIRECT_URL = "/"
 
-
 AUTH_CLASSES = [
     "iaso.api.auth.authentication.CsrfExemptSessionAuthentication",
     "rest_framework_simplejwt.authentication.JWTAuthentication",
 ]
 
+skip_polio_test = False
```
Comment (madewulf) [inline hat/settings.py:271]: This doesn't feel like a good solution, as we want to have other plugins and not to have to include code in settings.py for each one of them

**Comment by olethanh** [comment]
Maybe we can use this method ? https://stackoverflow.com/questions/49117482/django-executing-tests-for-app-not-in-installed-apps#comment110606562_58161401 ?

**Comment by SimonDus** [comment]
So, I moved the check of the loaded plugins into a tests_setting.py. 

When tests are skipped unit test notify that some tests were skipped which is quite nice. If you have a better idea to work around it, lemme know :)

image

**Comment by olethanh** [comment]
replaced by https://github.com/BLSQ/iaso/pull/573

**Pull request closed without merging**